pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/CovarianceAggregationFunction.java

```diff
@@ -55,6 +56,9 @@ public class CovarianceAggregationFunction implements AggregationFunction<Covari
   protected final boolean _isSample;
 
   public CovarianceAggregationFunction(List<ExpressionContext> arguments, boolean isSample) {
+    if (arguments.size() != 2) {
```

Comment:
Let's use `Precondition.checkArgument`. Please refer to `HistogramAggregationFunction`.

Reply:
Changed. Thanks.